diff --git a/src/kpip/_vendor/certifi/core.py b/src/kpip/_vendor/certifi/core.py
index 1c9661c..6bb2293 100644
--- a/src/kpip/_vendor/certifi/core.py
+++ b/src/kpip/_vendor/certifi/core.py
@@ -37,14 +37,20 @@ if sys.version_info >= (3, 11):
             # We also have to hold onto the actual context manager, because
             # it will do the cleanup whenever it gets garbage collected, so
             # we will also store that at the global level as well.
-            _CACERT_CTX = as_file(files("certifi").joinpath("cacert.pem"))
+            _CACERT_CTX = as_file(
+                files("kpip._vendor.certifi").joinpath("cacert.pem")
+            )
             _CACERT_PATH = str(_CACERT_CTX.__enter__())
             atexit.register(exit_cacert_ctx)
 
         return _CACERT_PATH
 
     def contents() -> str:
-        return files("certifi").joinpath("cacert.pem").read_text(encoding="ascii")
+        return (
+            files("kpip._vendor.certifi")
+            .joinpath("cacert.pem")
+            .read_text(encoding="ascii")
+        )
 
 else:
 
@@ -73,11 +79,11 @@ else:
             # We also have to hold onto the actual context manager, because
             # it will do the cleanup whenever it gets garbage collected, so
             # we will also store that at the global level as well.
-            _CACERT_CTX = get_path("certifi", "cacert.pem")
+            _CACERT_CTX = get_path("kpip._vendor.certifi", "cacert.pem")
             _CACERT_PATH = str(_CACERT_CTX.__enter__())
             atexit.register(exit_cacert_ctx)
 
         return _CACERT_PATH
 
     def contents() -> str:
-        return read_text("certifi", "cacert.pem", encoding="ascii")
+        return read_text("kpip._vendor.certifi", "cacert.pem", encoding="ascii")
